src/couch_epi_sup.erl

```diff
@@ -42,3 +48,36 @@ init([]) ->
         ?SUP(couch_epi_keeper_sup, [])
     ],
     {ok, { {one_for_one, 5, 10}, Children} }.
+
+%% ===================================================================
+%% Internal functions definitions
+%% ===================================================================
+
+start_plugins() ->
+    Plugins = application:get_env(couch_epi, plugins, []),
+    io:format(user, "PLUGS: ~p~n", [Plugins]),
+    ensure_started(Plugins).
+
+ensure_started(Apps) ->
+    start_applications(Apps, []).
+
+start_applications([], _Acc) ->
+    ok;
+start_applications([couch_epi|Apps], Acc) ->
+    start_applications(Apps, Acc);
+start_applications([App|Apps], Acc) ->
+    case not lists:member(App, Acc) of
+        true ->
+            case application:start(App) of
```

Comment:
I'm not comfortable with starting applications here, given that should be taken care of by the erlang release mechanism. I can't think of an alternative right now but we should find one.